java-jacoco/src/test/java/org/sonar/plugins/jacoco/JaCoCoOverallSensorTest.java

```diff
@@ -64,106 +59,106 @@ public void before() {
     configuration = mock(JacocoConfiguration.class);
     when(configuration.shouldExecuteOnProject(true)).thenReturn(true);
     when(configuration.shouldExecuteOnProject(false)).thenReturn(false);
-    context = mock(SensorContext.class);
-    fileSystem = new DefaultFileSystem((File)null);
+    context = SensorContextTester.create(new File(""));
+    fileSystem = context.fileSystem();
     fileSystem.setWorkDir(new File("target/sonar"));
     pathResolver = mock(PathResolver.class);
-    project = mock(Project.class);
     perspectives = mock(ResourcePerspectives.class);
     sensor = new JaCoCoOverallSensor(configuration, perspectives, fileSystem, pathResolver, javaResourceLocator, javaClasspath);
   }
 
   @Test
   public void testSensorDefinition() {
-    assertThat(sensor.toString()).isEqualTo("JaCoCoOverallSensor");
+    org.fest.assertions.Assertions.assertThat(sensor.toString()).isEqualTo("JaCoCoOverallSensor");
```

Comment:
assertThat is already part of the `static import`. No need to use the fully qualified name.